backend/zenao/v1/zenao_gnolit.pb.go

```diff
@@ -332,3 +332,42 @@ func (b *BatchProfileRequest) GnoLiteral(typePrefix string, linePrefix string) s
 	buf.WriteString("}")
 	return buf.String()
 }
+
+func (c *CreatePollRequest) GnoLiteral(typePrefix string, linePrefix string) string {
```

Comment:
Just for my own knowledge what is this about is this autogenerated ?

Reply:
this is useful to generate literals for realms initialization and `maketx run` content